code/package.lisp

```diff
@@ -2,6 +2,9 @@
   (:use
    #:cl)
 
+  (:local-nicknames
+   (#:a #:alexandria))
```

Comment:
Do we need to depend on all of Alexandria for only `when-let`? (I'm not very religious about keeping Spell dependency-free though.)

Reply:
There will be more uses and more substantial uses of `alexandria` later (as well as `closer-mop`). I hope that makes it acceptable.

Reply:
Yep, works for me.